### ISSUE TITLE
Provide alt text for the web alert [Should]

### ISSUE THREAD

**Issue by NatalieDavied** [department-of-veterans-affairs/va-mobile-feature-support#928]
# Description

We need to provide alt text for the web alert, so that Veterans who use screen readers have the same experience as a user who doesn't use a screen reader. 

This feedback was provided to the MFS team during Midpoint Review as should [feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/121289):

> With aligning to the web alert, the alert should provide alt text for the icon or screen reader only text, to inform the screen reader user of the alert and provide parity with the visual experience (sighted users parse both the color and the icon as information that should be paid attention to). For example, the screen reader might announce "Warning. Pay your balance now...". Note: there is currently a ticket open with the VADS team to update the component with this guidance. [WCAG 1.1.1 Non-text content.](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

<img width="260" height="115" alt="web alert" src="https://github.com/user-attachments/assets/133fd752-01fc-4b68-9bc3-f827043b10c0" />

# Additional Info and Resources

- This feedback has been considered vetted during a sync on 10/07.
- Adding @kimmccaskill-oddball's [comment](https://github.com/department-of-veterans-affairs/va-mobile-feature-support/issues/928#issuecomment-3638077192) here for visibility: 

> Related to Alerts, there is another a11y issue that while isn't related, could be worked on at the same time.
> https://github.com/department-of-veterans-affairs/va-mobile-app/issues/12107 came up which mentions that the collapsed state is not announced on first visit. The QA person mentioned there are multiple alerts in the app that have this issue so it should be fixed within the component itself and not within implementation.

# Acceptance Criteria
- [ ] The alert provides alt text for the icon or screen reader only text
- [ ] After completing this ticket add a note to the comment of [121289](https://github.com/department-of-veterans-affairs/va.gov-team/issues/121289).
- [ ] Update MFS [release notes](https://github.com/department-of-veterans-affairs/va-mobile-feature-support/tree/main/release-notes).

**Comment by mmiddaugh** [comment]
Per @rtwell, officially Matt and the Design System Team owns the fixes for these defects. His team would determine priority among other work in their backlog. Mobile team may be able to help in the interim. I'll add to Global Mobile board for review and consideration.
